tests/unit/type/list.tcl

```diff
@@ -60,8 +60,55 @@ start_server [list overrides [list save ""] ] {
         assert_equal [string repeat d 500] [r lindex list3 0]
     }
 
+    r debug quicklist-packed-threshold 5b
```

Comment:
we'll need to move this code into the "test" block and tag it with `needs:debug` (and the recovery code).
otherwise it'll fail when executed on external servers which block DEBUG.